### PR TITLE
[IMP] marketing automation: updated first_campaign

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3423041&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:21]: I know that you detail how to do this in the paragraphs below, but I think it would be helpful if there were 1-2 sentences explaining exactly what it means/what it does when you choose an audience for the campaign. Why is it significant?

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:43]: What are a few examples of the kinds of extra-specific configurations we can add here?

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (brse-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:56]:
```suggestion
In the rule fields, custom equations can be configured for Odoo to use when filtering who
```
Since this implies one or many

**Review by brse-odoo — COMMENTED** [review]
@ksc-odoo I just finished reviewing this PR, great job again! I had very few suggestions for this one - as always, feel free to ping me about them. Thanks!

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (ksc-odoo) [reply content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:43]: Well, this is a bit tricky. Since there are literally hundreds of random configurations, I don't want to list a few, and the reader wrongfully assumes they can't specify something they want to. So, for the sake of avoiding confusion, I'm going to leave this open-ended, but I fully understand the suggestion. Thanks for pointing this out.

**Comment by ksc-odoo** [comment]
Thanks for the always helpful review, @brse-odoo -- I made all the suggested edits, and believe it's ready for another quick look from you (whenever you get a chance)...Once you approve, I'll pass this along to the next stage. Thanks! 👍

**Review by brse-odoo — APPROVED** [review]

**Comment by brse-odoo** [comment]
> Thanks for the always helpful review, @brse-odoo -- I made all the suggested edits, and believe it's ready for another quick look from you (whenever you get a chance)...Once you approve, I'll pass this along to the next stage. Thanks! 👍

@ksc-odoo The changes look great, just approved! Ready to send to the next round

**Comment by ksc-odoo** [comment]
Thanks for the approval, @brse-odoo ! Okie dokie @meng-odoo -- this one's ready for a review from you now (whenever you get a chance) 👍

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:14]:
```suggestion
--> New` to reveal a blank campaign form.
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:46]:
```suggestion
Lastly on the campaign form is the :guilabel:`Filter` field. This is where a more specific
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:40]: Is this what the sentence meant to convey? To me it wouldn't make sense for Odoo not to process *any* records with a shared email address
```suggestion
   :guilabel:`Unicity based on` field so Odoo only processes one record for each customer email
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:50]: Apparently asterisks don't work inside of a guilabel, they just show up as asterisks :)
```suggestion
If left alone, the :guilabel:`Filter` field reads: :guilabel:`Match all records`. That means
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:69]:
```suggestion
   :guilabel:`Records` refer to contacts in the system that fit the specified
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:81]:
```suggestion
from the middle of the sentence ":guilabel:`Match records with all of the following rules`" to
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:90]:
```suggestion
the rule equation further define the criteria, which is used to determine which records in the
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (meng-odoo) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:94]:
```suggestion
rule, or click :guilabel:`New Rule` beneath the rule equation fields. When either are clicked, a new
```

**Review by meng-odoo — APPROVED** [review]
Hi @ksc-odoo, this doc looks great! I had a few small corrections, but please go ahead and push this to technical review after you've made those changes. Thank you!

**Comment by ksc-odoo** [comment]
Thanks for the feedback @meng-odoo 👍 I made all the suggested edits, and believe this one is now ready for technical review @samueljlieber 👍

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (StraubCreative) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:47]: Grammar and specificity
```suggestion
Last on the campaign form is the :guilabel:`Filter` field. This is where more specific targeting options can be layered into the campaign to further narrow the number and type of recipients that receive the marketing automation material.
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (StraubCreative) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:60]: Most of the equations are preset formulas, that are *customizable*
```suggestion
In the rule fields, customizable equations can be configured for Odoo to use when filtering who to include
```

File: content/applications/marketing/marketing_automation/getting_started/first_campaign.rst
Comment (StraubCreative) [inline content/applications/marketing/marketing_automation/getting_started/first_campaign.rst:74]: Not useful unless we include some more surrounding UI to provide a navigational queue as to its location. Would change or remove.

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 
Approving with comments.
The line break comments need to be fixed however everything else is optional, thanks!

@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
